ant/src/common/balances.lua

```diff
@@ -0,0 +1,59 @@
+local utils = require(".common.utils")
+local json = require(".common.json")
+
+local balances = {}
+
+function balances.walletHasSufficientBalance(wallet)
+	return Balances[wallet] ~= nil and Balances[wallet] > 0
+end
+
+function balances.info()
+	return {
+		Name = Name,
+		Ticker = Ticker,
+		TotalSupply = tostring(TotalSupply),
+		Logo = Logo,
+		Denomination = tostring(Denomination),
+		Owner = Owner,
+	}
+end
```

Comment:
this can be the response for the `Info` handler - i'm not sure why we need it in balances

Reply:
Fair point - originally I figured this would be good for unit testing, but we can use AOLoader for that.